src/rdo.rs

```diff
@@ -1271,19 +1271,18 @@ pub fn rdo_loop_decision<T: Pixel>(sbo: &SuperBlockOffset, fi: &FrameInvariants<
   let mut lrf_output;
   let mut cdef_input;
 
-  if fi.sequence.enable_cdef {
-    // all stages; reconstruction goes to cdef so it must be additionally padded
-    // TODO: use the new plane padding mechanism rather than this old kludge.  Will require
-    // altering CDEF code a little.
-    cdef_input = cdef_sb_padded_frame_copy(fi, sbo, &fs.rec, 2);
-    lrf_input = cdef_sb_frame(fi, &fs.rec);
-    lrf_output = cdef_sb_frame(fi, &fs.rec);
-  } else {
-    // no cdef; unpadded reconstruction offered directly to LRF optimization
-    cdef_input = cdef_empty_frame(&fs.rec);
-    lrf_input = cdef_sb_padded_frame_copy(fi, sbo, &fs.rec, 0);
-    lrf_output = cdef_sb_frame(fi, &fs.rec);
-  }
+  assert!(fi.sequence.enable_cdef);
+  // all stages; reconstruction goes to cdef so it must be additionally padded
+  // TODO: use the new plane padding mechanism rather than this old kludge.  Will require
+  // altering CDEF code a little.
+  cdef_input = cdef_sb_padded_frame_copy(fi, sbo, &fs.rec, 2);
+  lrf_input = cdef_sb_frame(fi, &fs.rec);
+  lrf_output = cdef_sb_frame(fi, &fs.rec);
+  // FIXME: T:T padded frame copy required for CDEF disabled path.
```

Comment:
So, CDEF disabled path will be handled in separate patch?
Then, CDEF cannot be disabled with this patch, correct?

Reply:
Yes, CDEF cannot be disabled after this patch. As the code was, the CDEF direction search is run even when CDEF is disabled. I'd like to follow up in a patch that enables toggling CDEF at runtime.

Reply:
Yeah,  the cdef disabling wasn't wired up to any speed level anyway.